addons/skin.estuary/xml/MyPVRGuide.xml

```diff
@@ -64,11 +64,19 @@
 						<left>300</left>
 						<right>60</right>
 						<height>30</height>
-						<label>[COLOR button_focus]$INFO[ListItem.StartTime][/COLOR]$INFO[ListItem.EndTime,[COLOR button_focus] - ,[/COLOR]] $INFO[ListItem.EpgEventTitle]$INFO[ListItem.EpisodeName, (,)]$INFO[ListItem.Genre,      [COLOR grey]$LOCALIZE[515]:[/COLOR] ]</label>
+						<label>[COLOR button_focus]$INFO[ListItem.StartTime][/COLOR]$INFO[ListItem.EndTime,[COLOR button_focus] - ,[/COLOR]] $INFO[ListItem.EpgEventTitle]$INFO[ListItem.Genre,      [COLOR grey]$LOCALIZE[515]:[/COLOR] ]</label>
+					</control>
+					<control type="label">
+						<top>29</top>
+						<left>300</left>
+						<right>60</right>
+						<height>30</height>
+						<textcolor>button_focus</textcolor>
+						<label>$VAR[GuideSeasonEpisodeLabel]</label>
```

Comment:
The new var and existing var `SeasonEpisodeLabel` look pretty much the same. Only difference is colors, right? You should make an include with parameters for the colors instead of copying xml.

Reply:
If you want to go with the grey, it will be identical except the bold on the episodename. Im not a fan of the grey, but i dont really care either way in the grand scheme of things.
Unfortunately ive no idea what you mean by an include parameter. If you can point me in the direction of where it is used that i can see how to implement it, happy to do that.

Reply:
Okay, the difference between the old and the new var is now only bold vs normal. So, you could make an include that takes bold vs normal as parameter. Look at PVR_Include.xml for examples on includes.